ui/lib/pki/addon/components/parsed-certificate-info-rows.ts

```diff
@@ -13,7 +13,30 @@ import { parsedParameterKeys } from 'vault/utils/parse-pki-cert-oids';
  *
  * @param {object} model - object of parsed attributes from parse-pki-cert util
  */
-export default class ParsedCertificateInfoRowsComponent extends Component {
+
+interface AttrsByKeys {
```

Comment:
I wonder if there's away to inline define these types below, leveraging the `const` below 💭

Reply:
Something like 
```js
interface AttrOptions {
label: string;
formatDate: string:
}

const attrsByKey: AttrOptions = {
....